apertium-uzb-kaa.uzb-kaa.dix

```diff
@@ -19553,6 +19553,4846 @@
+<e>       <p><l>Ikavonich<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Ikavonich<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Ikonich<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Ikonich<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Ilasov<s n="np"/><s n="cog"/></l>    <r>Ilasov<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Ilchenko<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Ilchenko<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Ilich<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Ilich<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Iliesku<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Iliesku<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Ilinchich<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Ilinchich<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Iliopolos<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Iliopolos<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Iliyasevich<s n="np"/><s n="pat"/></l>    <r>Iliyasevich<s n="np"/><s n="pat"/></r></p></e>
+<e>       <p><l>Illarionov<s n="np"/><s n="cog"/></l>    <r>Illarionov<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Ilminskaya<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Ilminskaya<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Ilminski<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Ilminski<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Ilyenko<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Ilyenko<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Ilyich<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Ilyich<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Imamovich<s n="np"/><s n="pat"/></l>    <r>Imamovich<s n="np"/><s n="pat"/></r></p></e>
+<e>       <p><l>Imsirevich<s n="np"/><s n="pat"/></l>    <r>Imsirevich<s n="np"/><s n="pat"/></r></p></e>
+<e>       <p><l>Imvich<s n="np"/><s n="pat"/></l>    <r>Imvich<s n="np"/><s n="pat"/></r></p></e>
+<e>       <p><l>Inakulesku<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Inakulesku<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Inamov<s n="np"/><s n="cog"/></l>    <r>Inamov<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Inchich<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Inchich<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Ingerson<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Ingerson<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Inoʻnu<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Inónu<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Inoyat<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Inoyat<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Inzko<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Inzko<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Ionesku<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Ionesku<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Ionov<s n
```

Comment:
The feminine forms should be removed; they'll be transferred automatically with the addition of the `<f>` label.